isis/src/base/apps/phocube/phocube.cpp

```diff
@@ -585,7 +604,32 @@ namespace Isis {
                 out[index] = pB[2];
                 index += 64 * 64;
               }
+              
+              // Compute the solar illumination mask
+              if ( sunilluminationmask ) {
+                  vector<double> sB(3), pB(3);
+                  cam->sunPosition(&sB[0]);
+                  cam->Coordinate(&pB[0]);
+
+                  // Look direction is the sun position to the surface point
+                  // in body-fixed
+                  vector<double> slookdir(3);
+                  vsub_c(&pB[0], &sB[0], &slookdir[0]);
+
+                  out[index]= (int) cam->target()->shape()->isVisibleFrom(sB, slookdir);
+                  index += 64 * 64;
+              }
+              
+              // Compute surface oblique detector resolution
+              if ( surfaceobliquedetectorresolution ) {
+                  double emission = cam->EmissionAngle() * DEG2RAD;
+                  if ( emission < HALFPI ) {
+                      out[index] = cam->DetectorResolution() / cos(emission);
+                  }
```

Comment:
Can this be replaced with the following?

`out[index] = cam->ObliqueDetectorResolution(useLocal=false);`

This should be the original math before it was changed to default to use the local emission

Reply:
@kledmundson Apologies for any confusion, the set of 4 lines I highlighted here can be replaced with the `ObliqueDetectorResolution` call. So this:
```C++
double emission = cam->EmissionAngle() * DEG2RAD;
if ( emission < HALFPI ) {
    out[index] = cam->ObliqueDetectorResolution(false) / cos(emission);
}
```

Should become this:
```C++
out[index] = cam->ObliqueDetectorResolution(false);
```

As the code in the camera function looks like this:
```C++
    if(!HasSurfaceIntersection()) {
      return Isis::Null;
    }

    double thetaRad;
    double emissionDeg;

    if(useLocal) {
      Angle phase, emission, incidence;
      bool success;

      LocalPhotometricAngles(phase, incidence, emission, success);
      emissionDeg = (success) ? emission.degrees() : Isis::Null;
    }
    else {
      emissionDeg = EmissionAngle();
    }

    thetaRad = emissionDeg*DEG2RAD;

    if (thetaRad < HALFPI) {
      return DetectorResolution()/cos(thetaRad);

    }

    return Isis::Null;
```

So if we run with `useLocal` set to false we run the following:
```C++
    if(!HasSurfaceIntersection()) {
      return Isis::Null;
    }

    double thetaRad;
    double emissionDeg;

    emissionDeg = EmissionAngle();
    thetaRad = emissionDeg*DEG2RAD;

    if (thetaRad < HALFPI) {
      return DetectorResolution()/cos(thetaRad);

    }

    return Isis::Null;
```